languages/tree-sitter-stack-graphs-javascript/src/stack-graphs.tsg

```diff
@@ -2418,7 +2433,10 @@ inherit .containing_class_value
 
 ;; #### Variables
 
-(primary_expression/identifier)@variable {
+[
+  (primary_expression/identifier)@variable
+  (member_expression . (identifier)@variable)
+] {
```

Comment:
This bit benefits from the ability to add edges multiple times. The grammar now renames `nested_identifier` (used in jsx constructs) to `member_expression` (which also exists as a production in `expression`), but the identifiers in the (originally `nested_identifier`) are _not_ expressions, and thus not processed by the `primary_expression/identifier` rule. So we have a situation where `member_expression`s may or may not be proper expressions with proper expressions as subnodes, and it's impossible to write a query isolating the no-actually-expressions identifiers, so there's a bit over overlap in the matches for the two cases here. Sigh...